File: LEETCODE-2834.js
/**
 * @param {number} n
 * @param {number} target
 * @return {number}
 */
var minimumPossibleSum = function(n, target) {
    let answer=0, len=0, number=1;
    hashmap={};
    while(len != n) {
        if(!hashmap[number]) {
            answer+= number;
            hashmap[target-number] = 1;
            len++;
        }
        number++;
    }
    return answer;
};

File: LEETCODE-2834.cpp
#include<bits/stdc++.h>
using namespace std;
class Solution {
public:
    long long minimumPossibleSum(int n, int target) {
        long long int answer = 0;
        int length = 0;
        unordered_map<int,int> mp;
        int number = 1;
        while(length != n) {
            if(mp[number] == 0) {
                answer+= number;
                mp[target-number] = 1;
                length++;
            }
            number++;
        }
        return answer;
    }
};